### PR TITLE
Refined Install Instructions

### PR DESCRIPTION
**Pull request by dylan-k**
---
name: Refined Install Instructions
about: Response to Issue #14 
---

# Pull request

Based on my install experience, and Issue #14 here are some potential clarifications.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -58,132 +58,82 @@
...
-
-4. The first thing we need to do is activate some library references
-5. Go to `Tools > References`
-
-![vba2](images/vba2.png)
-
-6. and you'll see this dialog box open. 
-7. Ensure all these items are selected if they are not, find them and select them. Namely the 2 necessary ones are
+2. Press & hold <kbd>Alt</kbd> then press <kbd>f11</kbd>. The Visual Basic Editor will open. You'll see something that looks like this: ![vba1](images/vba1.png)
+3. The first thing we need to do is activate some library references. Go to `Tools > References`. [vba2](images/vba2.png)
```
Comment (tallguyjenks) [inline README.md:107]:
```suggestion
3. The first thing we need to do is activate some library references. Go to `Tools > References`.  ![vba2](images/vba2.png)
```
missing `!` for image embedd :)

File: README.md [hunk 2]
```diff
@@ -58,132 +58,82 @@
...
+6. Next we need to get the code into the Outlook Application. For this, you'll need the `.vb` files located in the `VBA Utilities` folder of this repository. You'll copy/paste the code from these macros to use them in Outlook.
+7. To start, open the 'Insert Module' dropdown depicted below. Select `Module` then paste in the contents of a `.vb` file. Do this for each macro you want to use, but `USER_CONFIG` and `SaveUtilities` are required. ![vba4](images/vba4.png)
+8. You'll need to change the name of each new module. To do this, use the `Properties` window shown below. It should automatically be visible when you open the Editor with the hotkey combo from earlier. Name the modules exactly as you see listed below. (Names should match the original filenames, minus the extension so use `SaveMeeting` for what you pasted from `SaveMeeting.vb`)  ![vba5](images/vba5.png)
+9. Open the `USER_CONFIG` module and edit the `vaultPathToSaveFileTo` variable with  the absolute path where you want your files to be sent. You must use a trailing backslash at the end of the path `\`
```
Comment (tallguyjenks) [inline README.md:117]:
```suggestion
9. Open the `USER_CONFIG` module and edit the `vaultPathToSaveFileTo` variable with the absolute path where you want your files to be sent. You must use a trailing backslash at the end of the path `\`
```
white space :)

File: README.md [hunk 3]
```diff
@@ -58,132 +58,82 @@
...
-1. At the top of your Outlook application there will be a little down arrow icon and some other icons in the top left of the application. 
-
-![QA1](images/QA1.png)
-
-2. Click the down arrow and select the `More Commands` option
-
-![QA2](images/QA2.png)
+After installing the Outlook utilities, you can make them friendlier to use, thanks to the `Quick Access Toolbar in Outlook`.
 
+1. At the top of your Outlook application there will be a little down arrow icon and some other icons in the top left of the application.  ![QA1](images/QA1.png)
```
Comment (tallguyjenks) [inline README.md:127]:
```suggestion
1. At the top of your Outlook application there will be a little down arrow icon and some other icons in the top left of the application. ![QA1](images/QA1.png)
```

File: README.md [hunk 4]
```diff
@@ -251,6 +199,14 @@ See [RELEASES](https://github.com/tallguyjenks/Obsidian-For-Business/releases)
 
 See [CODE OF CONDUCT](CODE_OF_CONDUCT.md)
 
+### Contributors
+
+[Return To Top](#table-of-contents)
+
+None yet! But PR's welcome!
```
Comment (tallguyjenks) [inline README.md:206]: should edit this to add yourself :)

`[@username][1]` 

and at bottom of document

[1]: link to your github

File: README.md [hunk 5]
```diff
@@ -58,132 +58,82 @@
...
     - Microsoft VBScript Regular Expressions 5.5
-- For the TEMPLATE_VAULT
-  - Obsidian
 
 ### Recommended
 
 [Return To Top](#table-of-contents)
 
-- Latest Versions of Obsidian and Recommended Community Plugins
+It is recommended to use the latest version of Obsidian. The template vault includes  community plugins. These should be updated to their latest versions using Obsidian's settings, the first time you open the vault.
```
Comment (tallguyjenks) [inline README.md:88]:
```suggestion
It is recommended to use the latest version of Obsidian. The template vault includes community plugins. These should be updated to their latest versions using Obsidian's settings, the first time you open the vault.
```

**Review by tallguyjenks — CHANGES_REQUESTED** [review]
thanks for this! looks a lot nicer :)

**Comment by dylan-k** [comment]
Accepted all suggested edits and added to contributors list.

**Review by tallguyjenks — APPROVED** [review]

**Review by tallguyjenks — APPROVED** [review]

**Pull request merged**